src/app/hierarchical-grid/hierarchical-grid-validator-service-extended/hierarchical-grid-validator-service-extended.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="grid-wrapper">
+    <igx-hierarchical-grid #hierarchicalGrid height="570px" width="100%" [primaryKey]="'CustomerID'" (formGroupCreated)="formCreateHandler($event)"
+        [batchEditing]="true" [rowEditable]="true">
+        <igx-column field="CustomerID" [editable]="false"></igx-column>
+        <igx-column field="CompanyName" [editable]="true" required></igx-column>
+        <igx-column field="ContactName" [editable]="true" required>
+        </igx-column>
+        <igx-column field="ContactTitle" [editable]="true" required></igx-column>
+        <igx-column field="Phone" [editable]="true" required phoneFormat="^[^a-zA-Z]*$">
+            <ng-template igxCellValidationError let-cell='cell' let-defaultErr>
+                <ng-container *ngTemplateOutlet="cell.defaultErrorTemplate">
```

Comment:
defaultErrorTemplate is not exposed on the cell. Should be:
```
<ng-template igxCellValidationError let-cell='cell' let-defaultErr='defaultErrorTemplate'>
                <ng-container *ngTemplateOutlet="defaultErr" >
...
```
For all templates here.